source/fundamentals/linq.txt

```diff
@@ -28,6 +28,12 @@ strongly typed collections of objects by using language keywords and operators.
 The {+driver-short+} automatically translates LINQ queries into
 :manual:`aggregation operations </aggregation>`.
 
+.. important:: 
+
+   LINQ3 is the only LINQ provider available in the {+driver-long+}. If you have
+   manually configured your project to use LINQ2, you must change your provider to
+   LINQ3 to use this version of the driver.
+
```

Comment:
I had meant this to refer to whatever custom configuration some users might have done to stick with LINQ2 after LINQ3 became the default. Is there a better way to phrase this?

Reply:
The MongoClientSettings LinqProvider property has been removed.

If the client has code to configure the client settings to use LINQ2 it will no longer compile.

They need to remove that code.

Reply:
Thanks for clarifying!